service/tests/test_load.py

```
@@ -28,23 +75,171 @@ def test_persons_load():
+
+    statement = select(ResearchOutput).where(
+        ResearchOutput.id == research_output["uuid"]
+    )
     assert session.execute(statement).first() is None
+
+    lf = app.load.pure.research_outputs.transform(research_outputs)
+    df_iter = lf.collect_batches(chunk_size=1)
+
+    print("Created research outputs iterator")
+
+    for df_iter in df_iter:
```

Ruff (0.15.11)

[warning] Loop control variable `df_iter` overrides iterable it iterates

(B020)